client/src/reducers/HomeReducer.ts

```diff
@@ -18,7 +18,7 @@ const initialState: HomeReducerProps = {
   componentIndex: 0,
   coursesAdded: [],
   sections: [],
-  term: "",
+  term: "1",
```

Comment:
Why are we hardcoding term 1?

Reply:
I thought it would make sense to have a default value since we won't want it to be left as unselected, but what do you think?

Reply:
Right, yeah that's good.

Reply:
I think we should prevent users from selecting courses before selecting term

Reply:
Yeah right now we don't have much validation/error handling - this would be good for the future.